src/UITests/UILibrary/LiveMode.cs

```diff
@@ -1,17 +1,46 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using AccessibilityInsights.SharedUx.Properties;
 using OpenQA.Selenium.Appium.Windows;
+using UITests.Utilities;
 
 namespace UITests.UILibrary
 {
     public class LiveMode
     {
         WindowsDriver<WindowsElement> Session;
 
+        const string OpenFileToolBarAutomationID = "1001";
+        const string OpenFileFolderTextBoxAutomationID = "41477";
+        const string OpenFileFileTextBoxAutomationID = "1148";
+
         public LiveMode(WindowsDriver<WindowsElement> session)
         {
             Session = session;
         }
-        public bool OpenFile(string folder, string fileName) => false;
+
+        public bool OpenFile(string folder, string fileName)
+        {
+            if (!Session.FindAndClickByAutomationID(AutomationIDs.MainWinLoadButton)) return false;
```

Comment:
>FindAndClickByAutomationID [](start = 25, length = 26)

I'd tend to call it a programmer error if these are ever called in a context where they don't exist. Test should fail so that the code owner has to go fix it

Reply:
Again, the idea was to make the return value reflect success and leaving it up to the specific test to decide if a failure of this method was a failure of the test. Will change that style per my other comment :)